database/migrations/2020_11_18_101929_add_user_private_profile_key.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddUserPrivateProfileKey extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table("users", function (Blueprint $table) {
+            $table->boolean('private_profile')->default(false);
```

Comment:
```suggestion
            $table->boolean('private_profile')->default(false)->after('home_id');
```

Please specify where the column should be created. Otherwise it will be created after created_at and updated_at.